Error while installing

Command being run: `GDF_ROOT=${HOME}/miniconda3/envs/cudf mvn -DskipTests install`

Stack trace:

```
[INFO]  at scala.reflect.internal.Definitions$DefinitionsClass.ObjectClass(Definitions.scala:257)
[INFO]  at scala.reflect.internal.Definitions$DefinitionsClass.init(Definitions.scala:1390)
[INFO]  at scala.tools.nsc.Global$Run.<init>(Global.scala:1242)
[INFO]  at scala.tools.nsc.Driver.doCompile(Driver.scala:31)
[INFO]  at scala.tools.nsc.MainClass.doCompile(Main.scala:23)
[INFO]  at scala.tools.nsc.Driver.process(Driver.scala:51)
[INFO]  at scala.tools.nsc.Driver.main(Driver.scala:64)
[INFO]  at scala.tools.nsc.Main.main(Main.scala)
[INFO]  ... 6 more
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] XGBoost JVM Package ................................ SUCCESS [  1.289 s]
[INFO] xgboost4j .......................................... FAILURE [08:10 min]
[INFO] xgboost4j-spark .................................... SKIPPED
[INFO] xgboost4j-flink .................................... SKIPPED
[INFO] xgboost4j-example .................................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 08:12 min
[INFO] Finished at: 2019-02-11T17:21:51-08:00
[INFO] Final Memory: 21M/100M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.2.2:compile (default) on project xgboost4j: wrap: org.apache.commons.exec.ExecuteException: Process exited with an error: 240 (Exit value: 240) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :xgboost4j
(rapids_env) siddhaganju@siddhaganju:~/xgboost/jvm-packages$
```

This is the stack trace with the updated repo. Any further ideas?

--- a/README.md
+++ b/README.md
@@ -51,12 +51,26 @@ sudo apt install libnccl2 libnccl-dev
 Install the Java toolchain:
 ```bash
 sudo apt update
-sudo apt install openjdk-8-jdk maven cmake
+sudo apt install openjdk-8-jdk cmake
 echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823
 sudo apt update
 sudo apt install sbt
 ```
+
+Install maven:
+```bash
+export MAVEN_VERSION=3.6.0
+wget -q https://www-us.apache.org/dist/maven/maven-3/${MAVEN_VERSION}/binaries/apache-maven-${MAVEN_VERSION}-bin.tar.gz
+tar xzvf apache-maven-${MAVEN_VERSION}-bin.tar.gz -C /opt
+ln -s /opt/apache-maven-${MAVEN_VERSION} /opt/maven
+```
+and add the following to your `.bashrc`:
+```bash
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+export PATH=/opt/maven/bin:${PATH}
+```
+
 ### Building XGBoost
 
 From your root source directory (e.g. `${HOME}/src`), run:

